test/statsCases/child-compiler-apply-entry-option/webpack.config.js

```diff
@@ -0,0 +1,20 @@
+"use strict";
+
+var TestApplyEntryOptionPlugin = require("./TestApplyEntryOptionPlugin");
+
+/** @type {import("../../../").Configuration} */
+module.exports = {
+	entry: {
+		parent: "./parent"
+	},
+	output: {
+		filename: "[name].js"
+	},
+	plugins: [
+		new TestApplyEntryOptionPlugin({
+			entry: {
+				child: "./child"
+			}
+		})
+	]
+};
```

Comment:
Maybe add `stats: { children: true, entrypoints: true }` here to see the child compilation stats too